tests/services/test_auth.py

```diff
@@ -66,3 +75,10 @@ def test_clear_credentials_causes_auth_api_to_be_called_on_subsequent_calls(
         auth.clear_credentials()
         auth(mock_request)
         assert mock_v3_conn.get.call_count == 2
+
+
+class TestCustomJWTAuth(object):
+    def test_call_returns_jwt_string(self, mock_custom_auth_function):
```

Comment:
This test sig should start with `test_get_credentials_`()

Reply:
changed.